client/components/Dashboard/RecentScenarios.jsx

```diff
@@ -87,25 +91,26 @@ const RecentScenarios = () => {
         >
           View my scenarios
         </Button>
-        <Button size="small" href="/scenarios">
+        <Button as="a" href="/scenarios">
```

Comment:
This should remain `size="small"` in order to ensure it's the same size as the other two buttons (right now this one has 14px font, while the other two have 13px)